### ISSUE TITLE
How to adjust learning rate for larger batch size 

### ISSUE THREAD

**Issue by Yuzz1020** [Alibaba-MIIL/ImageNet21K#17]
Hi, 

Thank you for your great work! I am currently trying to run your code with a larger batch size to better parallelize the code. Could you provide some hints on how to adjust the learning rate with respect to the larger batch size? I noticed that in the DeiT paper, the authors propose to adjust the learning rate with the same scale of batch size, should I also do such scaling? 

Thank you for your time!

**Comment by mrT23** [comment]
The answer is a bit tricky.
some initial assumptions:
1. transformed are harder to train than conv nets, and are more sensitive to hyperparameters (including lr)
2. there is a major difference in the optimal learning rate when you train a network from scratch, compared to when you fine-tune it from a previous pretraining.
3. if you don't have a lots (!) of GPU resources, hard to do hyperparameters tuning and searching on imagenet 21K.

given 1-3, the scheme I suggested in the article is to initially train a net on imagenet1K, and then transfer it to imagenet21K for 80 epochs. Since this is transfer learning, I used regular adam with lr=3e-4. i believe that this scheme is quite robust, on TResNet-m i trained with a large batch (416*8).  with ViT the batch was smaller (64*8).

you can try other learning rates, I think a reasonable range for adam optimizer is 2e-4 to 5e-4.
DeiT paper talks about training from scratch, which is different. notice that they suggest 1000 epochs !

**Comment by Yuzz1020** [comment]
Thank you for the fast response! I'll try with the lr you suggested.
